.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Yes, it's a copy from CBA website.

Reply:
What about other pages, mind adding that info to the Description?

Reply:
Comments are not allowed in `.json` files, so I'll add notes in github.

Reply:
Meeting-note: We will updated description of Github PR.

Reply:
yes, I removed `min-width:0` from cell classes, removed media queries and edited 1 linter and 1 syntax errors.

Reply:
> removed media queries 
Why?

>  I removed min-width:0 from cell classes
Why?

Reply:
> 
> 
> > removed media queries
> 
> Why?
properties are for `main` and `footer`. We don't have footer yet and don't have ready `main` section. Maybe we should bring it back when they'll be ready for styling but now that style would be redundant and confusing I think. It's for simplicity in short terms.

> > I removed min-width:0 from cell classes
> 
> Why?
Because we don't have that Galaxy S5 issue in case of Privacy Manager website header. (See https://github.com/cmints/website/issues/8 )

Reply:
I see, where the decision is coming from, thanks for detailed explanation.

> properties are for main and footer. We don't have footer yet and don't have ready main section. Maybe we should bring it back when they'll be ready for styling but now that style would be redundant and confusing I think. It's for simplicity in short terms.

The idea is to have a generic minimal styles for the website(s), as most of our websites will have eventually those sections, make sense to have it here as well.

> Because we don't have that Galaxy S5 issue in case of Privacy Manager website header. (See cmints/website#8 )

I think it's because we don't have `<pre>` or any other tag or style that causes that issue, thinking it's the matter of adding content to experience that issue, even though I don't exactly remember what exactly causes the issue, it seems like this was in the library intentionally. I think we either need to understand and fix the bug, rather than just invalidating it from the library.

Let's treat this file as sensible defaults and make decision according whether it make sense to have specific styles as part of whole website, because in future we will have those sensible defaults on a separate repository and add them to our projects like: `npm i cmints-sensible-defaults` and load them into project in less files accordingly.

Reply:
Meeting-notes:
- Narek will bring back other missing classes.

Reply:
Done.